ClientServer.go

```diff
@@ -47,7 +46,10 @@ func Server() {
 		// send new string back to client
 		_, sendingerror := conn.Write([]byte(newmessage + "\n"))
```

Comment:
yes I have just checked it , if I use ' = ' instead of ' := ' then it will be fine